SR2020/2019-12-18-competition-date-announcement.md

```diff
@@ -0,0 +1,18 @@
+---
+to: sr2020-teams
+subject: SR2020 Competition Date Announcement
+---
+
+Hello There!
+
+As an early Christmas present to you all, today's the day we confirm the date for the Student Robotics 2020 competition!
+
+The competition will be held from *Saturday 18th April* until *Sunday 19th April*. The competition will be held at the University of Reading, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed.
```

Comment:
Maybe remove the 2nd "The competition will be held" and make it one sentence?

Reply:
I like it!